Kommunikasjon/Mottaker.php

```diff
@@ -79,12 +79,17 @@ public static function clean(String $string, String $allow_pattern = 'A-Za-z0-9-
      */
     public static function cleanMobil(String $mobil)
     {
-
+        
         // REMOVE FACEBOOK 3 SPECIAL CHARS
         if (strlen($mobil) == 11 && (int)$mobil == 0) {
             $mobil = substr($mobil, 3);
         }
 
+        // Remove country code
```

Comment:
@mariusmandal Jeg la til denne, men ser nå at Sveve kanskje støtter landskode uten + og 00 i front uten noe mer. Formening om den bør vekk igjen?

Reply:
Vi skal (kanskje) inn med landskode-støtte etter hvert med den nye id-løsningen @AsgeirSH, men det ligger litt frem i tid, og krever noe omskriving uansett, så du kan godt ta de ut inntil videre (for det mener jeg vi gjør i dag 😊)

Reply:
Den er grei, da merger jeg og kvalitetssikrer master-branch i ettermiddag 😄